aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -218,5 +219,22 @@ bool quantized_equal(const Tensor& self, const Tensor& other) {
   return 0 == memcmp(self_data, other_data, self.numel() * self.element_size());
 }
 
+/* Calculate the quantization params for the activation tensor */
+std::tuple<double, int64_t> _choose_qparams(const Tensor& self, bool reduce_range) {
+  at::Tensor a;
+  auto input_contig = self.contiguous();
+  float x_min = input_contig.min().item<float>();
+  float x_max = input_contig.max().item<float>();
+
+  auto q_params = quant_utils::ChooseQuantizationParams(
+        /*min=*/x_min,
+        /*max=*/x_max,
+        /*qmin=*/0,
+        /*qmax=*/255,
+        reduce_range=reduce_range);
```

Comment:
really? can you pass keyword arguments in C++?

Reply:
Ah no. It was supposed to be a comment. I'll update the PR